extensions/test-extension/eslint.config.js

```diff
@@ -0,0 +1,8 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+
+module.exports = [
+  {
+    files: ["**/*.ts"],
```

Comment:
The config this is replacing references @bentley/imodeljs-recommended, which may be a different config, will have to take a look

Reply:
You were correct in the change to use itwinjs-recommended config, renamed in the past it appears